### DISCUSSION TITLE
Cannot get this to install using self-hosted instructions

### DISCUSSION THREAD

**Discussion by risharde** [firefly-iii/firefly-iii#7267] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi, I had a lot of hopes to get this installed today but following the instructions here: https://docs.firefly-iii.org/firefly-iii/installation/self-hosted/?mtm_campaign=docu-internal&mtm_kwd=self_hosted

I get a non-helpful error on the command line:
[root@blah ff3]# composer create-project grumpydictator/firefly-iii --no-dev firefly-iii 5.7.18
Creating a "grumpydictator/firefly-iii" project at "./firefly-iii"                                                                                                                 
  [InvalidArgumentException]                                                                                                
  Could not find package grumpydictator/firefly-iii with version 5.7.18 in a version installable using your PHP version, P  
  HP extensions and Composer version.

I have php 8.2 installed and the modules mentioned in the documentation.

I also tried: composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii 6.0.5

Same error. I've even tried downgrading to 7.4. No go. 

There must be an easier way to get this to self host!

### Debug information

Couldn't reach this point.

**Comment by JC5** [comment]
Make sure you have the latest version of composer by using `composer selfupdate`.

**Reply by risharde** [reply]
Thank you for the response! I ended up using docker but I will try to test this as well
